spec/models/user_spec.rb

```diff
@@ -399,6 +399,15 @@
         expect(User.active).not_to include(user3)
       end
 
+      it "returns users who have confirmed their account" do
```

Comment:
Since we're trying to test that newsletters aren't sent to unconfirmed accounts, how about adding a test for `Newsletter#list_of_recipient_emails` instead? https://github.com/consul/consul/blob/master/spec/models/newsletter_spec.rb#L50